mod/primitives/pkg/encoding/ssz/vector.go

```diff
@@ -100,6 +100,11 @@ func (v Vector[T]) Elements() []T {
 	return v
 }
 
+// ItemLength returns the length of the item in the VectorBasic.
```

Comment:
comment is misleading to me, this is the number of bytes required to represent the root element of a vector, not the size of its items